### DISCUSSION TITLE
pdf merge says 'Invalid form data: no form file found for extensions: [.pdf]'

### DISCUSSION THREAD

**Discussion by VijayKumarVV** [gotenberg/gotenberg#553] [Q&A]
can you please help here.

curl --location --request POST 'http://localhost:3000/forms/pdfengines/merge' \
--form '=@"/D:/burstIQ/pdf1.pdf"' \
--form '=@"/D:/burstIQ/pdf2.pdf"' \
--form 'pdfFormat="PDF/A-1a"'

**Comment by NiklasBr** [comment]
You probably need the form identification names (`files`), like this:

```curl
curl --request POST 'http://localhost:3000/forms/pdfengines/merge' \
--form 'files=@"/D:/burstIQ/pdf1.pdf"' \
--form 'files=@"/D:/burstIQ/pdf2.pdf"' \
--form 'pdfFormat="PDF/A-1a"'
```
